packages/turbo-gen/src/generators/custom.ts

```diff
@@ -11,16 +11,18 @@ export async function generate({
   project,
   opts,
 }: CustomGeneratorArguments) {
+  let isOnboarding = false;
   let generators = getCustomGenerators({ project, configPath: opts.config });
   if (!generators.length) {
-    logger.error(`No custom generators found.`);
+    logger.error(`No generators found.`);
     console.log();
 
     const { answer } = await prompts.confirm({
-      message: `Would you like to add generators to ${project.name}?`,
+      message: `Would you like to add an example generator to ${project.name}?`,
```

Comment:
```suggestion
      message: `Would you like to add an custom generator to ${project.name}?`,
```

If we're going with "custom" generators, let's do that everywhere?